ProgrammingFundamentals/Session1/Assignment1/testAssignment1.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * 
+ * @author Vinod Patel
+ *
+ */
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class TestAssignment {
+
+	first obj = new first();
+	
+	//to test the actual result and expected result	
+	@Test
+	public void maintest() {
+		int binary = obj.binary_to_octal(11001000);

```

Comment:
binary_to_octal is static method so you can directly call by class name no need to create object of the class for calling this method. This line is throwing error in your test.